filament/src/details/Engine.cpp

```diff
@@ -396,6 +402,10 @@ void FEngine::shutdown() {
     ASSERT_PRECONDITION(ThreadUtils::isThisThread(mMainThreadId),
             "Engine::shutdown() called from the wrong thread!");
 
+#if FILAMENT_ENABLE_FGDBG
+    delete debug.fgdbg;
+#endif
```

Comment:
`matdbg` does not `delete debug.server` here. But I think we should

Reply:
@prideout ?

Reply:
Why do you need to delete fgdbg is you're using a smart pointer?  Can you rename `server` to `matdbg` to help avoid confusion between the two debuggers?  You can also convert matdbg to use a smart pointer so that it is consistent with the new fgdbg stuff.

Reply:
Done! 
Yes it used to be a raw pointer, this is outdated code